gamedata/alldefs_post.lua

```diff
@@ -263,7 +263,7 @@ function UnitDef_Post(name, uDef)
 	if uDef.customparams then
 		if not uDef.customparams.techlevel then uDef.customparams.techlevel = 1 end
 		if not uDef.customparams.subfolder then uDef.customparams.subfolder = "none" end
-
+		addeddemon = false
```

Comment:
?

Reply:
very likely, but that would be another problem with more possible issues. Might get to it one day but not right now.